easybuild/easyconfigs/j/jupyterlab/jupyterlab-1.1.1-CrayGNU-19.10-batchspawner.eb

```
@@ -110,22 +114,22 @@ exts_list = [
     ('colorcet', '2.0.2'),
     ('zstandard', '0.13.0'),
     ('itk-core', '5.0.1', {
-        'source_tmpl': 'itk_core-%(version)s-cp36-cp36m-manylinux1_x86_64.whl',
+        'source_tmpl': 'itk_core-%(version)s-cp38-cp38-manylinux1_x86_64.whl',
```

```suggestion
        'source_tmpl': 'itk_core-%(version)s-cp%(pymajver)s%(pyminver)s-cp%(pymajver)s%(pyminver)s-manylinux1_x86_64.whl',
```
Ca we use %(pymajver)s%(pyminver)s here and below @twrobinson ? or is it not the python version?